hardware/tftp/Android.mk

```
@@ -1,6 +1,6 @@
 LOCAL_PATH := $(call my-dir)
 
-ifneq ($(filter sdm660 msm8998 sdm845 sm6125 sm8150,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter sdm660 msm8998 sdm845 sm6125 sm8150 sm8250,$(TARGET_BOARD_PLATFORM)),)
```

```suggestion
ifneq ($(filter sdm660 msm8998 sdm845 $(TRINKET) sm8150 $(KONA),$(TARGET_BOARD_PLATFORM)),)
```

Let's stick with and eat our own dogfood.

---

Anyway, why are we still carrying this completely useless and annoying guard? All platforms supported on R branches need these symlinks...

---

Unnecessary style change. Just go for it as it is.

Perhaps one day we may just remove the barriers entirely, as they may make no sense anymore, like you suggested.

---

Let's remove them at some point...